### DISCUSSION TITLE
GSA Mod Function Request - DinoColoring Brush Use - Support for All Creature Colors

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#580] [Suggestions]
As it stands, it appears that the current iteration of bot the ASE and ASA mod's paint brush function host colors 1-100, leaving out color IDs 128-254. Given that these are not unrestricted colors and can be obtainable in-game, it would be optimal to include these in the UI of the DinoColoring brush. 

[https://ark.wiki.gg/wiki/Color_IDs](https://ark.wiki.gg/wiki/Color_IDs)
![image](https://github.com/gameserverapp/Platform/assets/44717221/8cc7cc94-bc0b-481c-89d8-0b0bb21c5eb9)

**Comment by mvaessen** [comment]
hi, all colors available in ARK should now be included in the paint brush!
